webui/react/src/utils/shortcut.ts

```diff
@@ -20,6 +34,8 @@ export const shortcutToString = (shortcut: KeyboardShortcut): string => {
   shortcut.meta && s.push(os.includes('Mac') ? 'Cmd' : os.includes('Win') ? 'Win' : 'Super');
   shortcut.shift && s.push('Shift');
   shortcut.alt && s.push('Alt');
-  shortcut.key && s.push(shortcut.key);
+  shortcut.key &&
+    !['Control', 'Meta', 'Alt', 'Shift'].includes(shortcut.key) &&
+    s.push(shortcut.key);
```

Comment:
using `.toUpperCase()` in #7184 to be case insensitive since Shift key being pressed could change case of `key` value.

Reply:
Added a util function `formatKey` to handle the case conversion and other staff